src/main/java/ch/njol/skript/expressions/ExprScript.java

```diff
@@ -42,8 +42,9 @@ public class ExprScript extends SimpleExpression<Script> {
 
 	static {
 		Skript.registerExpression(ExprScript.class, Script.class, ExpressionType.SIMPLE,
-			"[the] [current] script",
-			"[the] script[s] [named] %strings%",
+			"[the] [current] script [file]",
+			"[the] script[s] [file[s]] [named] %strings%",
+			"[the] skript file[s] %strings%",
```

Comment:
skript file is a lot more than a script
```suggestion
			"[the] [current] script [file]",
			"[the] script[s] [file[s]] [named] %strings%",
```